syntax = "proto3";

package users;

// The Users service definition.
service UsersService {
  rpc GetUsers(UserRequest) returns (UserResponse);
}

// The request message for the GetUsers RPC.
message UserRequest {}

// The response message containing the user message.
message UserResponse {
  string message = 1;
}